README.md

```diff
@@ -1,84 +1,93 @@
-# nf-core/nascent
+# ![nf-core/nascent](docs/images/nf-core-nascent_logo_light.png#gh-light-mode-only) ![nf-core/nascent](docs/images/nf-core-nascent_logo_dark.png#gh-dark-mode-only)
 
-**Nascent Transcription Processing Pipeline**.
+[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/nascent/results)[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.XXXXXXX)
 
-[![Build Status](https://travis-ci.com/nf-core/nascent.svg?branch=master)](https://travis-ci.com/nf-core/nascent)
-[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.32.0-brightgreen.svg)](https://www.nextflow.io/)
-[![DOI](https://zenodo.org/badge/157735234.svg)](https://zenodo.org/badge/latestdoi/157735234)
```

Comment:
This is the DOI for Nascent - should be added back :-) 

[![DOI](https://zenodo.org/badge/157735234.svg)](https://zenodo.org/badge/latestdoi/157735234)

Reply:
there is a main doi so to say that points to the whole thing, and then one per release, we are usually using the main one for the badges as the don't change between releases 
<img width="369" alt="Screenshot 2022-10-20 at 00 00 33" src="https://user-images.githubusercontent.com/12273093/196812916-4b1a9c51-28fb-475a-ab7d-3ca62c754ac0.png">